utils/notification.ts

```diff
@@ -29,6 +31,11 @@ export const showNotification = (
   message: string | null,
   params: any = notificationTypes.info
 ): void => {
+  if (params === notificationTypes.danger) {
+    consola.error('[Notification Error]', message)
+    captureMessage(String(message))
+    return
```

Comment:
We should not throw an error here, because this function would only display the toast for some specific message. Usually, errors are thrown outside this function. `try catch` is unnecessary here.

Reply:
It seems no need to return the result of `captureMessage`

Reply:
just so we can remove blank empty `return` statemennt

Reply:
> It seems no need to return the result of `captureMessage`

Aight I'm still unused to see empty return

Reply:
> just so we can remove blank empty return statement (or return console.log instead)
> Aight I'm still unused to see empty return

It's used to stop executing function here. The return type of showNotification function is also `void`. Let's move on~